_includes/header.html

```diff
@@ -61,15 +62,16 @@
 						</hgroup>
 
 						<div class="download">
-	  						<a href="{{ site.tgmpa.github }}/archive/{{ site.tgmpa.version }}.zip" class="latest-zip">
-								<img border="0" width="90" alt="Download latest release zipfile" src="{{ site.tgmpa.zipimage }}">
-							</a>
-	  						<a href="{{ site.tgmpa.github }}/archive/{{ site.tgmpa.version }}.tar.gz" class="latest-tar">
-								<img border="0" width="90" alt="Download latest release tarball" src="{{ site.tgmpa.tarimage }}">
-							</a>
+							<p class="download-icon">
+								<a href="{{ '/download/' | prepend: site.tgmpa.url }}" title="Download the latest release">
```

Comment:
Ok, removing it is then. My front-end accessbility knowledge is a bit out of date it seems ;-)

> Title attributes should not be used except on iframe elements.

Though this sounds out of date too - title *should* be used on input elements which have the HTML5 `pattern` attribute set as that will allow the browser to display as an error hint.
http://www.w3.org/TR/2011/WD-html5-20110525/common-input-element-attributes.html#the-pattern-attribute